backend/app.ts

```diff
@@ -12,7 +14,7 @@ import { MatchReporter } from './match-reporter/MatchReporter'
 const jsonParser = bodyParser.json()
 const urlencodedParser = bodyParser.urlencoded({ extended: false })
 
-const app = express()
+const app: Application = express()
```

Comment:
i'd expect timescript to be ablt to infer the time of express() - isn't that the case? 🤔

Reply:
you are right